react/svg.native.js

```diff
@@ -0,0 +1,7 @@
+import Svg from 'react-native-svg';
```

Comment:
Minor: "External dependencies" DocBlock. We use this in the other template files at least.

Reply:
The use of a dependency here makes this a bit more difficult / less obvious to consume. Anything which uses this must now also define `react-native-svg` as a dependency, which we don't mention anywhere.

Reply:
Only if they are running in RN mode correct?

Reply:
Yes, I suppose so.